### ISSUE TITLE
Cramped Prompt

### ISSUE THREAD

**Issue by ShawnieBoy** [magarena/magarena#210]
With some of the prompt text being quite large at times, would there be a possibility of having the prompt/player pane stretch to accommodate? More for the vertical bar.

(Land Tax)
![land tax](https://cloud.githubusercontent.com/assets/2363180/7215476/175f4e42-e5d2-11e4-8c3c-8b0ca8030fc6.png)

**Comment by lodici** [comment]
Makes sense. Personally, I would prefer this Yes/No prompt to appear center screen - _user you have a decision to make and this is it!_ - easier on the eyes.

**Comment by ShawnieBoy** [comment]
That's true - Although that always has the possibility of obscuring the battlefield state

**Comment by lodici** [comment]
Yes it would need to be moveable or hideable or both. Perhaps minimize down to what is now the action prompt.

**Comment by ShawnieBoy** [comment]
>Yes it would need to be moveable or hideable or both. Perhaps minimize down to what is now the action prompt.

That makes sense, yes
